pkg/apiserver/debugapi/endpoint/param_model_def.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package endpoint
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+
+	"github.com/thoas/go-funk"
+)
+
+var APIParamModelText = NewAPIParamModel("text")
+
+var APIParamModelBool = NewAPIParamModel("bool")
```

Comment:
Why we don't have transformers for bool? (like making 1, 2, 3, 4, 5 to true)

Reply:
Nice catch. There's a bug.